examples/Makefile

```diff
@@ -1,7 +1,7 @@
 # http://www.apache.org/licenses/LICENSE-2.0.txt
 #
 #
-# Copyright 2016 Intel Corporation
+# Copyright 2017 Intel Corporation
```

Comment:
@aarontay It's general comment, you shouldn't change the year in license header if you just edit the file. If you add a new file then the year in license header should indicate current year.